tests/test_assets/gx/checkpoint_result_fail.json

```diff
@@ -64,7 +64,7 @@
                         }
                     },
                     {
-                        "success": false,
+                        "success": true,
```

Comment:
What does changing this value do?

Reply:
This is a test asset representing a checkpoint result. Here I am changing one of the individual expectation results from a flat out fail to a success with one failed value which is now recognized as something to raise a warning about. This is so I can evaluate the code's ability to distinguish failures and warnings and report out both in the unit tests in test_gx.py.

Reply:
Thanks for the explanation!